proto/v1/event.proto

```diff
@@ -67,9 +67,14 @@ message EventCollectionRequest {
   // - discoveryDate
   // - occurrenceTime
   string date = 3;
+  // The property used for filtering
+  string filter_prop = 4;
```

Comment:
Since an event can have multiple prop vals that have to do with severity (e.g., directInjuries and directDeaths), do we only want to allow filtering by one prop?

Reply:
Do you foresee the use case of filtering by multiple property? I wonder if a combination of filtering in mixer (reduce data size) and in client (for UI actions) is preferred. Anyway, if we do need to filter by multiple props, can update this into a nested structure.

Reply:
I think this depends on how many severity props events will have and how much we want to allow a user to play around with setting filters. Having just one filter is ok for now